src/material/divider/divider.scss

```diff
@@ -1,16 +1,27 @@
-$width: 1px;
+@use '../core/tokens/token-utils';
+@use '../core/tokens/m2/mat/divider' as tokens-mat-divider;
+
+
 $inset-margin: 80px;
 
 .mat-divider {
+  @include token-utils.create-token-values(
+    tokens-mat-divider.$prefix, tokens-mat-divider.get-unthemable-tokens());
+
   display: block;
   margin: 0;
-  border-top-width: $width;
   border-top-style: solid;
+  @include token-utils.use-tokens(
+    tokens-mat-divider.$prefix, tokens-mat-divider.get-token-slots()) {
+    @include token-utils.create-token-slot(border-top-color, color);
+    @include token-utils.create-token-slot(border-top-width, thickness, true);
```

Comment:
You can drop the `, true` since the width token is guaranteed to be defined because it's part of the unthemable tokens.